src/library/simplified/tag/criteo.js

```diff
@@ -0,0 +1,472 @@
+//~~tv:20010.20140827
```

Comment:
Nice to meet you! I believe so, yes. I just added three files so there would be nothing to compare.